internal/cli/atlas/quickstart/prompt.go

```diff
@@ -89,3 +95,31 @@ func newClusterProviderQuestion() *survey.Question {
 		},
 	}
 }
+
+func newMongoShellQuestion(clusterName string) *survey.Confirm {
+	return &survey.Confirm{
+		Message: fmt.Sprintf("Do you want to access %s with MongoDB Shell?", clusterName),
+		Help:    mongoshHelp,
+	}
+}
+
+func newMongoShellQuestionProvidePath() *survey.Confirm {
+	return &survey.Confirm{
+		Message: "No MongoDB shell version detected. Do you want to provide the path to your MongoDB shell binary?",
```

Comment:
[nit] from seeing this in action I think we can split this message, lets do a `fmt.Printl("No MongoDB shell version detected")` and then this question as 
```suggestion
		Message: "Do you want to provide the path to your MongoDB shell binary?",
```

I would even print that message of no shell detected to stderr instead of stdout but that last part is optional